app/helpers/month_names_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module MonthNamesHelper
```

Comment:
Hm, not sure if I agree with the pattern here. Helpers should be attached to controllers, IMO. I'd like your opinion on this, though.

Reply:
I had my doubts regarding the best place for this helper, as I haven't worked with them much. What would you suggest?

Reply:
Putting the helper method under the helper attached to a specific controller (in this case ProduceItemHelper)

Reply:
I moved it to ProduceItemHelper, and deleted the MonthNamesHelper